docs/docsite/rst/community/steering/community_topics_workflow.rst

```diff
@@ -0,0 +1,126 @@
+..
+   THIS DOCUMENT IS OWNED BY THE ANSIBLE COMMUNITY STEERING COMMITTEE. ALL CHANGES MUST BE APPROVED BY THE STEERING COMMITTEE!
+   For small changes (fixing typos, language errors, etc.) create a PR and ping @ansible/steering-committee.
+   For other changes, create a `community topic <https://forum.ansible.com/new-topic?category=project&tags=community-wg>`_ to discuss them.
+   (Creating a draft PR for this file and mentioning it in the community topic is also OK.)
+
+.. _community_topics_workflow:
+
+Ansible community topics workflow
+=================================
+
+Overview
+--------
+
+This document describes the Ansible community topics workflow (hereinafter ``Workflow``) to provide guidance on successful resolving topics in the asynchronous way.
+
+The Workflow is a set of actions that need to be done successively within the corresponding time frames.
+
+.. note::
+
+   If you have any ideas on how the Workflow can be improved, please create an issue in this repository or pull request against this document.
+
+Creating a topic
+----------------
+
+Any person can `create a topic <https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=community-wg>`_ tagged with ``community-wg`` under the ``Project Discussions`` category in the `Ansible Forum <https://forum.ansible.com/>`_. A :ref:`Steering Committee member<steering_members>` can tag the forum post with `community-wg-nextmtg` to put it on the meeting agenda.
+
+Workflow
+========
+
+.. note::
+
+  This is a rough scenario and it can vary depending on a topic's complexity and other nuances, for example, when there is a mass agreement up-front.
```

Comment:
```suggestion
  This is a rough scenario and it can vary depending on a topic's complexity and other nuances, for example, when there is a mass agreement upfront.
```